.github/workflows/release-production.yml

```diff
@@ -20,6 +31,13 @@ jobs:
       - name: Get tag
         run: echo "TAG=${GITHUB_REF#refs/*/v}" >> $GITHUB_ENV
 
+      - name: Set default tag
```

Comment:
This step isn't needed if it's a version tag. You should remove or else you'll break the release flow

Reply:
Updated.